apps/src/lib/ui/accounts/LtiRosterSyncSettings.jsx

```diff
@@ -16,6 +18,14 @@ export default function LtiRosterSyncSettings(props) {
   const [changed, setChanged] = useState(false);
 
   const handleSubmit = () => {
+    const eventPayload = {
+      lms_type: props.lmsType,
```

Comment:
Nit: this is really the lms_name not type

Reply:
You're right, I'm was going off Ali's doc. Will connect with him regarding naming of these values.